markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+  config_profile_contact = '<your_name> (<your_github_handle>)'
+  config_profile_url = 'https://<institutional_url>.com'
+  max_memory = 2.TB
+  max_cpus = 128
+  max_time = 720.h
+  igenomes_base = '/<path>/<to>/igenomes/'
+}
+
+process {
+  executor = 'sge'
+  queue = { task.cpus > 24 ? 'big' : 'small' }
+  maxRetries = 2
+  clusterOptions = { '-l h_vmem=${task.memory.toGiga()}G' }
+  beforeScript = 'module load singularity'
+}
+```
+
+For a full list of `process` directives, please see the [Nextflow documentation](https://www.nextflow.io/docs/latest/process.html#process-directives)
+
+#### executor scope
+
+The executor scope allows the use of further executor _specific_ options that are inbuilt into Nextflow. You should check the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html#scope-executor) to see what options are available for your respective scheduler.
+
+One sometimes useful option for smaller clusters with less sophisticated fair-use management is the `queueSize` directive. This allows you to specify the maximum number of jobs of a given Nextflow run can submit in parallel at any one time. So to prevent Nextflow from swamping a (small) cluster thousands of jobs at once and blocking the cluster for other users, you can limit this as follows:
+
+```nextflow
+params {
+  config_profile_description = '<your_cluser_name> cluster profile provided by nf-core/configs.'
+  config_profile_contact = '<your_name> (<your_github_handle>)'
+  config_profile_url = 'https://<institutional_url>.com'
+  max_memory = 2.TB
+  max_cpus = 128
+  max_time = 720.h
+  igenomes_base = '/<path>/<to>/igenomes/'
+}
+
+process {
+  executor = 'sge'
+  queue = { task.cpus > 24 ? 'big' : 'small' }
+  maxRetries = 2
+  clusterOptions = { '-l h_vmem=${task.memory.toGiga()}G' }
+  beforeScript = 'module load singularity'
+}
+
+executor {
+  queueSize = 8
+}
+```
+
+Where a given Nextflow run can only have 8 submitted jobs at once (and will wait to one job is completed before submitting the next one).
+
+A similar directive is the `submitRateLimit` which specifies how many jobs can be specified in a given time frame (as some clusters may penalise you for over-submitting)
+
+```nextflow
+params {
+  config_profile_description = '<your_cluser_name> cluster profile provided by nf-core/configs.'
+  config_profile_contact = '<your_name> (<your_github_handle>)'
+  config_profile_url = 'https://<institutional_url>.com'
+  max_memory = 2.TB
+  max_cpus = 128
+  max_time = 720.h
+  igenomes_base = '/<path>/<to>/igenomes/'
+}
+
+process {
+  executor = 'sge'
+  queue = { task.cpus > 24 ? 'big' : 'small' }
+  maxRetries = 2
+  clusterOptions = { '-l h_vmem=${task.memory.toGiga()}G' }
+  beforeScript = 'module load singularity'
+}
+
+executor {
+  queueSize = 8
+  submitRateLimit = '10 sec'
+}
+```
+
+Where a maximum of 10 jobs can be submitted per second
+
+#### <your_container\> scope
```

Comment:
I'll go for singularity/docker/podman scope here directly.
Or maybe just container scope in the title, and explain that it's either singularity, docker, podman, or even conda...